my_game.toml

```diff
@@ -17,6 +17,10 @@ PLAYER_SPEED_LIMIT = 5  # px/update
 PLAYER_INVINCIBILITY_SECONDS = 5  # secs
 PLAYER_FIRE_RATE = 0.2  # secs
 PLAYER_SHOT_RANGE = 400  # px
+PLAYER_THRUST_KEY = 65362
+PLAYER_FIRE_KEY = 32
+PLAYER_TURN_RIGHT_KEY = 65363
+PLAYER_TURN_LEFT_KEY = 65361
 
```

Comment:
Please add a link to the key numbers here: https://api.arcade.academy/en/latest/arcade.key.html#key
Also add comments so a human can understand the current settings:
_PLAYER_FIRE_KEY = 32 # Space_

Reply:
Ok

Reply:
Im not sure i understand

Reply:
Add comments in the config file. That way, other humans can understand it better.

Reply:
Ok that makes sense